SemDiff.Core/GitHub.cs

```diff
@@ -118,11 +134,29 @@ public Task UpdateLimitAsync()
             return await response.Content.ReadAsStringAsync();
         }
 
+        /// <summary>
+        /// Gets each page of the pull request list from GitHub.
+        /// Once the list is complete, get all the pull request files for each pull request.
+        /// </summary>
+        /// <returns>List of pull request information.</returns>
         public async Task<IList<PullRequest>> GetPullRequestsAsync()
         {
             //TODO: Investigate using the If-Modified-Since and If-None-Match headers https://developer.github.com/v3/#conditional-requests
             var url = $"/repos/{RepoOwner}/{RepoName}/pulls";
-            var pullRequests = await HttpGetAsync<IList<PullRequest>>(url);
+            var paginationPRs = await HttpGetAsync<IList<PullRequest>>(url);
+            var pullRequests = paginationPRs;
+            while(paginationPRs != null)
+            {
+                paginationPRs = null;
+                if(paginationURL != null)
+                {
+                    paginationPRs = await HttpGetAsync<IList<PullRequest>>(paginationURL);
+                    foreach(var cur in paginationPRs)
```

Comment:
There is a `pullRequest.AddRange(paginationPRs)` method that we could use if `pullRequest` was a `List<T>`. That could be a good reason for a change. (Since I think Json.NET creates a `List<T>` anyway)